### DISCUSSION TITLE
Instrument Showcase

### DISCUSSION THREAD

**Discussion by DJthefirst** [SammyIAm/Moppy2#153] [Show and tell]
I am starting this thread so makers can showcase their instruments. I recently completed my Pneumatic instrument based on a video I saw by NTSB. I plan on making a tutorial of sorts on YT when I can find time but for now, here is a video of it playing.

[https://youtu.be/zDYBKL4m5AE](url)
[![Pnumatic Valves](https://img.youtube.com/vi/zDYBKL4m5AE/0.jpg)](https://www.youtube.com/watch?v=zDYBKL4m5AE)

It works by chopping the air up into sound waves. It is also extremely effective reaching 70-75 dB at 20psi and 100-110 dB at 60psi I have not tested it at 100psi yet. I also wrote a PWMController class that I will make a merge request for soon that might be useful for speakers, standard Dc motors, and my valves that don't require dir commands. should allow all 16 channels to be played by an Uno. I'm also working on my Large TC so that should be done sometime next year.

**Comment by samphonic** [comment]
Well _that's_ super cool!!  What kind of valves are those?  (Edit: specifically, I'd have guessed normal solenoid valves would be too slow for this, are they something special?) I tried to make a calliope back in the day and it never worked, but I bet if you attached those to some tubes/horns you could get some pretty cool sounds.

**Reply by DJthefirst** [reply]
*Sorry for the late reply. They are Smc SX10 high-speed pneumatic valves. They can operate up to 1200hz at 80W a valve.

**Reply by samphonic** [reply]
Wow!  Those are neat!  Going to have to keep those in mind for future projects...

**Comment by DJthefirst** [comment]
### Tesla Coil, Air Compressor, and Hammered Dulcimer

It's been a hot minute since my last post but I've finished a few projects and started some more. First off I don't have a good video but the air compressor has been fitted with a light-up muffler that is connected to the played notes. I also finished my large 4ft tesla coil which can reach 4-5ft archs. My newest project is a hammered dulcimer which I plan on fitting solenoids to make self-playing.

**Air Compressor**
https://www.youtube.com/shorts/3zybAmYjuns
https://youtube.com/shorts/IvIAESyWKVQ?feature=share

**Tesla Coil**
https://youtu.be/VixODGMAFFc
https://youtu.be/Y-uaI5LF-Es

**Dulcimer**
![image](https://user-images.githubusercontent.com/37967524/171458201-f124d13c-31ae-49a6-90da-16f287c7ee2b.png)

**Comment by DJthefirst** [comment]
Update on the Ducilmer: 
I finished an initial prototype in about 4 hrs waiting on parts to complete the build.
https://youtu.be/vrS-SjBaq7E

**Comment by samphonic** [comment]
Well that's gotta be one of the coolest things I've seen in terms of MIDI-controlled instruments.  It sounds great!

Those are the same little solenoids I used trying to build my handbell rig.  I love that they come with little nuts on the end already; perfect for striking (though too loud for bells it turns out, so I switched to nylon).

**Comment by Oldmanspub** [comment]
ready for this year retro exhibition, brszk.hu
![IMG_4275](https://github.com/Sammy1Am/Moppy2/assets/122678108/d0db062f-358e-41b5-871e-d37490df18d6)
![IMG_4277](https://github.com/Sammy1Am/Moppy2/assets/122678108/5de916c0-3e27-4a17-968b-1d79b1aa1d5a)
![IMG_4298](https://github.com/Sammy1Am/Moppy2/assets/122678108/d5d4c1e2-997e-4f96-9e43-4710bddac3f8)

**Reply by DJthefirst** [reply]
👏🏻

**Reply by samphonic** [reply]
Ooo!  Love that you got that many floppy drives all mounted inside an actual PC case.
